apigw-http-api-lambda-ts-cdk/README.md

```diff
@@ -0,0 +1,70 @@
+# Amazon API Gateway HTTP API with AWS Lambda integration
+
+This pattern in CDK offers a boilerlate to generate an Amazon API Gateway HTTP API endpoint with "ANY" method from the specified path. The Lambda function provided in TypeScript only returns a dummy "HelloWorld".
+
+If Amazon API Gateway HTTP API (v2) covers all the features' requirements, it's recommended to use HTTP API (v2) over REST API (v1). The HTTP API has the advantage to be significantly less expensive, and it peforms better.
```

Comment:
Can you quantify the savings and performance? As written, the statement is vague.